src/router/server.js

```diff
@@ -0,0 +1,79 @@
+import express from 'express';
+import axios from 'axios';
+import cors from 'cors';
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+const EXTERNAL_API_BASE_URL = 'https://api.escuelajs.co/api/v1/products';
```

Comment:
We are using different API: `https://fakestoreapi.com`

https://fakestoreapi.com/docs

Reply:
Oh yeah, I forgot to change that.